### PR TITLE
Add /offline as overview-page for offline magazine

### PR DESCRIPTION
**Pull request by FluidSense**
We have an "archive" of old offlines at old, and would like for this to be present at OWF as well.
This is nice to have if you want to view old offlines without navigating the frontpage.

### CODE REVIEW

File: src/offline/index.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+import { getOfflines, getRemainingOfflines } from 'frontpage/api/offline';
+import { IOfflineIssue } from 'frontpage/models/Offline';
+import OfflineList from './components/OfflineList';
+import Spinner from 'common/components/Spinner';
+
+export const Offline: React.FC = () => {
+  const [offlines, setOfflines] = useState<IOfflineIssue[]>([]);
+
+  /** Get the first batch of Offlines for feast loading */
```
Comment (oleast) [inline src/offline/index.tsx:11]: Loading is a feast 🥳 🎈

Comment (FluidSense) [reply src/offline/index.tsx:11]: Hahaha, I shamelessly copied this file from frontpage for faster development, but it should probably be refactored to share as much as possible with frontpage. Noticed the comment there, but forgot to fix it so now it is present in two files 😂

Comment (oleast) [reply src/offline/index.tsx]: Yeah, I had i feeling I was the original sinner in this case 🙈

**Comment by FluidSense** [comment]
Opened an issue (#513) to redo this view, so that prokom can refer to it in Offline and we can rather revisit this, the _agile_ way

**Review by aslakhol — APPROVED** [review]

**Pull request merged**